verification/curator-service/ui/src/components/Sidebar/index.tsx

```diff
@@ -52,7 +52,7 @@ const Sidebar = ({ drawerOpen }: SidebarProps): JSX.Element => {
         [user],
     );
 
-    const diseaseName = process.env.REACT_APP_DISEASE_NAME ?? 'COVID-1922222';
+    const diseaseName = process.env.REACT_APP_DISEASE_NAME ?? 'COVID-19';
```

Comment:
actually my comment above about `process.env` makes me think this isn't working well either :(